ql/src/java/org/apache/hadoop/hive/ql/txn/compactor/CompactorUtil.java

```diff
@@ -133,6 +150,24 @@ public static List<Partition> getPartitionsByNames(HiveConf conf, String dbName,
     }
   }
 
+  public static Database resolveDatabase(HiveConf conf, String dbName) throws MetaException, NoSuchObjectException {
+    try {
+      return getMSForConf(conf).getDatabase(MetaStoreUtils.getDefaultCatalog(conf), dbName);
+    } catch (NoSuchObjectException e) {
+      LOG.error("Unable to find database {}, {}", dbName, e.getMessage());
+      throw e;
+    }
+  }
+
+  public static Table resolveTable(HiveConf conf, String dbName, String tableName) throws MetaException {
+    try {
+      return getMSForConf(conf).getTable(MetaStoreUtils.getDefaultCatalog(conf), dbName, tableName);
+    } catch (MetaException e) {
+      LOG.error("Unable to find database {}, {}", dbName, e.getMessage());
```

Comment:
This method is resolving for table, This log should be `Unable to find Table {}, {}`, tableName

Reply:
Nice catch. Modified accordingly. Done.